data/json/items/classes/comestible.json

```diff
@@ -6,8 +6,8 @@
     "name": { "str": "Spice" },
     "//": "Spices are inedible in their base form but useful in crafting",
     "flags": [ "NUTRIENT_OVERRIDE" ],
-    "weight": "5 g",
-    "volume": "250 ml",
+    "weight": "4 g",
+    "volume": "1000 ml",
```

Comment:
250 ml would be 131.87 grams.  Looks like most store bottles of spices are around 50 grams.

Reply:
Good web resource find, I wish I had found that when I started on this ticket.  Using https://www.aqua-calc.com/calculate/food-volume-to-weight as a resource I redid the table that I posted in the original issue with a wider range of spices.  Here are the results:

Volume (mL) | Mass (g) | Density (g/mL) | Charge volume (mL) | Charge volume | Mass per charge (g) | Spice
-- | -- | -- | -- | -- | -- | --
1000 | 527.5 | 0.5275 | 5 | 1 tsp | 2.6375 | cinnamon web
1000 | 466.63 | 0.46663 | 5 | 1 tsp | 2.33315 | ground black pepper
1000 | 1217.3 | 1.2173 | 5 | 1 tsp | 6.0865 | table salt
1000 | 365.19 | 0.36519 | 5 | 1 tsp | 1.82595 | oregano
1000 | 541 | 0.541 | 15 | 1 Tbsp | 8.115 | chili powder
1000 | 358.43 | 0.35843 | 5 | 1 tsp | 1.79215 | cayenne pepper
1000 | 101.44 | 0.10144 | 15 | 1 Tbsp | 1.5216 | basil
1000 | 135.26 | 0.13526 | 15 | 1 Tbsp | 2.0289 | sage
1000 | 655.99 | 0.65599 | 15 | 1 Tbsp | 9.83985 | garlic
1000 | 290.8 | 0.2908 | 15 | 1 Tbsp | 4.362 | thyme
1000 | 1052.46 | 1.05246 | 5 | 1 tsp | 5.2623 | mustard

This analysis provides a slightly different outcome of 4.16 grams per charge (my original analysis had 4.26).  As you can see from the table there is some significant variability in the mass per charge between spices, but the 4 gram selection still seems to be reasonably valid as a single abstract value for an arbitrary spice.

I will agree that spices coming in 1 liter containers seems really high.  My cabinet (which represents shopping in the south-east United States) has 3 main spice container sizes; 88 mL (a small metal box), 553 mL (a plastic cylinder), and 1L (a plastic rectangular prism). 

For the purposes of abstraction would it feel more realistic if I changed the volume to 500 mL and the number of charges to 50?

Reply:
Do you think most recipes use 4.16 grams of spice when they use a charge?  My assumption would be that most recipes spice requirements cap out at a teaspoon or two and each teaspoon would probably be 2 grams?  So maybe 500 ml with 100 charges and each charge has a weight of 2.08 grams?  But anything you choose is an improvement at this point.  Let me know once you decide and I'll remove the request for changes :)

Reply:
Based on Maleclypse's comments I did an additional set of recipe reviews.  Based on that review I have modified my assumption of the charge volume to be 1 tsp for all of the in game spices instead of a mixture of 1 tsp and 1 Tbsp.

Using this new charge volume assumption I have redone the analysis once again using just the in game spices.  

Volume (mL) | Mass (g) | Density (g/mL) | Charge volume (mL) | Charge volume | Mass per   charge (g)
-- | -- | -- | -- | -- | --
1000 | 358.43 | 0.35843 | 5 | 1 tsp | 1.79215 | cayenne pepper
1000 | 527.5 | 0.5275 | 5 | 1 tsp | 2.6375 | cinnamon
1000 | 426 | 0.426 | 5 | 1 tsp | 2.13 | curry powder
1000 | 466.63 | 0.46663 | 5 | 1 tsp | 2.33315 | black pepper
1000 | 1217.3 | 1.2173 | 5 | 1 tsp | 6.0865 | salt
1000 | 568 | 0.568 | 5 | 1 tsp | 2.84 | italian seasoning
1000 | 973 | 0.973 | 5 | 1 tsp | 4.865 | seasoning salt
1000 | 1052.46 | 1.05246 | 5 | 1 tsp | 5.2623 | mustard powder

Based on this I have modified the mass per charge to 3500 mg (3.5 g), and since I have changed assumed charge volume I have also adjusted the volume for 100 charges to be 500 mL.

I have rerun the automated tests, as well as repeated the manual tests details in the issue.

Reply:
Not sure of the etiquette here, so the originator close the conversation or sub the responder who believes they have resolved it?